sklearn/feature_selection/_from_model.py

```diff
@@ -99,8 +100,15 @@ class SelectFromModel(MetaEstimatorMixin, SelectorMixin, BaseEstimator):
         ``threshold`` in the case where the ``coef_`` attribute of the
         estimator is of dimension 2.
 
-    max_features : int, default=None
-        The maximum number of features to select.
+    max_features : int, callable, default=None
+        The maximum number of features to select. ``max_features`` may be a
+        whole number representing the number of maximum features to allow,
+        or a callable that takes only ``X`` as an argument and returns the
+        number of maximum features to allow. When ``max_features`` is a
+        callable, the maximum number of features is calculated as
+        ``max_features_=max_features(X)``. For example, if one wanted to
```

Comment:
We do not need to describe `max_features_` here. We need to describe what the input of the callable is.